packages/cozy-client/src/associations/HasManyFiles.js

```diff
@@ -37,11 +51,12 @@ export default class HasManyFiles extends HasMany {
       // Photos always have a datetime field in metadata
       const lastDatetime = lastRelDoc.attributes.metadata.datetime
       // cursor-based pagination
-      const cursorKey = [this.target._type, this.target._id, lastDatetime]
-      const startDocId = relationships[relationships.length - 1]._id
-      const cursorView = [cursorKey, startDocId]
+      const cursor = newCursor(
+        [this.target._type, this.target._id, lastDatetime],
+        relationships[relationships.length - 1]._id
+      )
```

Comment:
Nice for making it generic :+1:

Reply:
I had to otherwise the type checker would complain :sweat_smile:

Reply:
Ah :D Well, it's still nice!